conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/ethereum/EthereumOperations.scala

```diff
@@ -0,0 +1,115 @@
+package tech.cryptonomic.conseil.indexer.ethereum
+
+import cats.effect.{Concurrent, Resource}
+import com.typesafe.scalalogging.LazyLogging
+import fs2.Stream
+import slickeffect.Transactor
+
+import tech.cryptonomic.conseil.common.config.Platforms.EthereumBatchFetchConfiguration
+import tech.cryptonomic.conseil.common.rpc.RpcClient
+import tech.cryptonomic.conseil.common.ethereum.EthereumPersistence
+import tech.cryptonomic.conseil.common.ethereum.rpc.EthereumClient
+import tech.cryptonomic.conseil.indexer.config.{Custom, Depth, Everything, Newest}
+
+/**
+  * Ethereum operations for Lorre.
+  *
+  * @param ethereumClient Ethereum client instance
+  * @param persistence Ethereum persistence instance
```

Comment:
Can I suggest to refrain from describing params like this? We might try to explain what each component does.
Otherwise I'd recommend leaving the desc empty

Reply:
Please check it now

Reply:
awesome